New version: MusicBrainz.Picard version 2.90000.0

vedantmgoyal9 opened the pull request:
### Pull request has been created with [WinGet Automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) :rocket:
 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115192&drop=dogfoodAlpha

vedantmgoyal9 committed:
New version: MusicBrainz.Picard version 2.90000.0

Trenly commented:
@wingetbot run

Trenly commented on manifests/m/MusicBrainz/Picard/2.90000.0/MusicBrainz.Picard.installer.yaml:
```diff
@@ -0,0 +1,24 @@
+- Architecture: x64
+  InstallerUrl: https://github.com/metabrainz/picard/releases/download/release-2.9/MusicBrainz-Picard-2.9.msix
+  InstallerSha256: 8F194CBB35B8FE6BEDBA9E7B204B9C6CB95C0464699023F8A531FA902A16D0DA
+  PackageFamilyName: MetaBrainzFoundationInc.org.musicbrainz.Picard_s89a1zkh1p1gp
+- Architecture: x64
+  InstallerUrl: https://github.com/metabrainz/picard/releases/download/release-2.9/MusicBrainz-Picard-2.9.msix
+  InstallerSha256: 8F194CBB35B8FE6BEDBA9E7B204B9C6CB95C0464699023F8A531FA902A16D0DA
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 2.8.5
+    ProductCode: MusicBrainz Picard
```
```suggestion
```

Trenly committed:
Update manifests/m/MusicBrainz/Picard/2.90000.0/MusicBrainz.Picard.installer.yaml

Trenly commented:
@wingetbot run

Scavy committed:
Corrected the version number, so that it matches the released build and should be able to go through verification.

Scavy commented:
I have made a PR with the fix to @vedantmgoyal2009 PR on his fork.
[https://github.com/vedantmgoyal2009/winget-pkgs/pull/490](https://github.com/vedantmgoyal2009/winget-pkgs/pull/490)
Since I can't do it here, and are not supposed to post a new PR here as long as this is active.
In case of no action from any of the participants, this PR is essentially blocking the release of a correct PR.

vedantmgoyal9 committed:
Merge pull request #490 from Scavy/patch-MusicBrainz.Picard-1

Patch music brainz.picard 1

vedantmgoyal2009 commented:
@Scavy your PR has been merged 👍

Scavy committed:
Update MusicBrainz.Picard.installer.yaml with needed fields

Added the needed fields, and made sure add anything that could be missing, by recreating the yaml file from WinGetCreate

Scavy committed:
Update MusicBrainz.Picard.installer.yaml

Scavy commented:
@vedantmgoyal2009 I've made yet a PR with the missing fields.
Didn't realize in the first PR that there would be fields missing as well. :)

vedantmgoyal9 committed:
Merge pull request #493 from Scavy/MusicBrainz.Picard-2.90000.0-6FA7F03842C79862F86023D1C4B614CA18216794-1

Music brainz.picard 2.90000.0 6 fa7 f03842 c79862 f86023 d1 c4 b614 ca18216794 1
